setup_wordpress.sh

```diff
@@ -1,3 +1,103 @@
 #!/bin/bash
+set -e
+echo '################## LEMP Stack Installation ####################'
+cd ~
+echo "------------------ Updating APT ------------------"
+echo
+sudo apt -y update && sudo apt -y upgrade
```

Comment:
why do we run `apt -y upgrade` here?

Reply:
I thought installing the updated packaged would be necessary.

Reply:
not a mistake.